DOCKER_ADVANCED.md

```diff
@@ -40,34 +40,20 @@ docker run -it --rm --env-file=path/to/setenv ghcr.io/openbb-finance/openbbtermi
 
 ## Display
 
-In order to display plots in the docker container, we can configure our XServer.
+In order to display plots in the docker container, we can use IPC socket to connect Desktop.
 
-Users familiar with Docker and X-Server can set the `DISPLAY` variable in the file [setenv](/docker/setenv) described
-above. If you use this approach remember to add `:0` at the end of your inet address. E.g. `DISPLAY=192.168.1.155:0`.
-
-### X-Server
-
-For help setting up the X-Server, I will go through this now:
-
-#### Setting up X Quartz/X11
-
-0. Install X Quartz from <https://www.xquartz.org/>
```

Comment:
we still do need x11 don't we?
for `xhost +` to work

Reply:
Because Linux Desktop was built with X11.
For remote user in Linux, common case is using SSH X-forwarding.
Run command , and it likes :

```bash
ssh -X example@example.com 'desktop-app-command'
```

In Mac/Windows , you need Quartz/Xming to do this. [1] 

1. https://kb.iu.edu/d/bdnt - Use X forwarding on a personal computer to securely run graphical applications installed on IU's research supercomputers